listenbrainz/webserver/static/css/follow.less

```diff
@@ -47,8 +47,16 @@
   }
 }
 
+.card-user-sn {
+  margin-bottom: 20px;
+}
+
 .follower-following-pills {
-  padding-bottom: 10px;
+  padding: 20px 0px;
+
+  button {
+    margin: 0px 6px !important;
```

Comment:
As much as possible we try to stay away from `!important`, but after starting to write a whole paragraph, I realized looking into the page code that the pills styles are applied directly to the element, not with a class.
Unfortunate, but so it goes.
These are the only good reasons to use `!important` :)